custom.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/grupozap/aegir/internal/pkg/utils"
+	livr "github.com/k33nice/go-livr"
+)
+
+func neq(args ...interface{}) livr.Validation {
+	not_allowed := utils.FirstArg(args...)
+
+	return func(value interface{}, builders ...interface{}) (interface{}, interface{}) {
```

Comment:
Why not returning type `error` here?
```suggestion
	return func(value interface{}, builders ...interface{}) (interface{}, error) {
```

Reply:
Closing because I didn't notice the return of `livr.Validation` is this way. Sorry.